src/main/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedAndReceivedPersonalisation.java

```diff
@@ -72,6 +72,8 @@ private String buildMrnDetails(MrnDetails mrnDetails) {
     }
 
     private String buildYourDetails(Appeal appeal) {
+        String phone = appeal.getAppellant().getContact().getPhone() != null ? appeal.getAppellant().getContact().getPhone() : appeal.getAppellant().getContact().getMobile();
```

Comment:
Create a common method that returns the phone number to use when a Contact is passed into it and result for the Appellant too (line 136 + 166).  Also I would suggest reversing the condition to be ```getPhone() == null``` (slightly quicker I think, also in other places in this PR but those read better as they return ```null```).